packages/commons/test/BundledAxeScriptProviderTest.cs

```diff
@@ -0,0 +1,24 @@
+using NUnit.Framework;
+using Deque.AxeCore.Commons;
+using Moq;
+using FluentAssertions;
+using System;
+
+namespace Deque.AxeCore.Commons.Test
+{
+    [TestFixture]
+    [NonParallelizable]
```

Comment:
Why does this have to be non-parallelizable?

Reply:
It could be parallelizable; will remove